<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPropertyGenerator</class>
 <widget class="QWidget" name="QtPropertyGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Property Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add Property</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Class name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lClassName"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cInline">
       <property name="text">
        <string>inline</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cGroup">
       <property name="text">
        <string>group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGenerate">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>770</width>
        <height>512</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>766</width>
            <height>508</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="pLayout"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>770</width>
        <height>512</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Source</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="pGetHeader">
           <property name="text">
            <string>Get Header Code</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="headerP"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="pGetSource">
           <property name="text">
            <string>Get Source Code</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="sourceP"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
